index.bs

```diff
@@ -2778,6 +2832,278 @@ The [=remote end steps=] with <var ignore>session</var> and |command parameters|
+      |serialized start node|, |realm| and |session|.
+
+   1. [=list/Append=] |start node| to |context nodes|.
+
+1. Assert [=list/size=] of |context nodes| is greater than 0.
+
+1. Let |type| be |locator|["<code>type</code>"].
+
+1. In the following list of conditions and associated steps, run the first set 
+   of steps for which the associated condition is true:
+
+   <dl>
+
+      <dt>|type| is the string "<code>css</code>"
+      <dd>
+         1. Let |selector| be |locator|["<code>value</code>"].
+
+         1. Let |result nodes| be a result of [=trying=] to [=locate nodes using css=]
+            given |current context target|, |context nodes|, |selector|, |maximum returned
+            nodes| and |session|.
+
+      <dt>|type| is the string "<code>xpath</code>"
+      <dd>
+         1. Let |selector| be |locator|["<code>value</code>"].
+
+         1. Let |result nodes| be a result of [=trying=] to [=locate nodes using xpath=]
+            given |current context target|, |context nodes|, |selector|, |maximum returned
+            nodes| and |session|.
+
+      <dt>|type| is the string "<code>innerText</code>"
+      <dd>
+         1. Let |selector| be |locator|["<code>value</code>"].
+
+         1. If |locator| [=map/contains=] <code>maxDepth</code>, let |max depth| be 
+            |locator|["<code>maxDepth</code>"]. Otherwise, let |max depth| be null.
+
+         1. If |locator| [=map/contains=] <code>ignoreCase</code>, let |ignore case| be 
+            |locator|["<code>ignoreCase</code>"]. Otherwise, let |ignore case| be false.
+
+         1. If |locator| [=map/contains=] <code>matchType</code>, let |match type| be 
+            |locator|["<code>matchType</code>"]. Otherwise, let |match type| be "full".
+
+         1. Let |result nodes| be a result of [=trying=] to [=locate nodes using inner text=]
+            given |current context target|, |context nodes|, |selector|, |max depth|,
+            |match type|, |ignore case|, |maximum returned node count| and |session|.
+
+1. Assert |maximum returned node count| is null or [=list/size=] of |result nodes| is less
+   than or equal to |maximum returned node count|.
+
+1. If |command parameters| [=map/contains=] "<code>serializationOptions</code>",
+   let |serialization options| be |command parameters|["<code>serializationOptions</code>"].
+   Otherwise, let |serialization options| be a [=/map=] matching the 
+   <code>script.SerializationOptions</code> production with the fields
+   set to their default values.
+
+1. If |command parameters| [=map/contains=] "<code>ownership</code>",
+   let |result ownership| be |command parameters|["<code>ownership</code>"].
+   Otherwise, let |result ownership| be "none".
+
+1. Let |serialized nodes| be the result of [=serialize as a remote value=] with
+   |result nodes|, |serialization options|, |result ownership|, a new map as
+   <var ignore>serialization internal map</var>, |realm| and |session|.
```

Comment:
Not sure this should be in `<var>` since it's not a variable (I can believe we do this wrong elsewhere in the spec however :) )

Reply:
I see other places where this type of language is required without the use of `<var>`. Corrected.